src/test/java/com/cloudlabs/server/account/AccountControllerTest.java

```diff
@@ -0,0 +1,80 @@
+package com.cloudlabs.server.account;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import com.cloudlabs.server.security.auth.dto.LoginDTO;
+import com.cloudlabs.server.user.User;
+import com.cloudlabs.server.user.UserRepository;
+import com.cloudlabs.server.user.dto.UserDTO;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+@TestInstance(Lifecycle.PER_CLASS)
+@WithMockUser(username = "tester", roles = { "USER" })
+public class AccountControllerTest {
+    @Autowired
+    protected MockMvc mockMvc;
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+
+    @Test
+    void retrieveUserDetails() throws Exception {
+        User mock_user = new User("tester", "tester", "tester@gmail.com", passwordEncoder.encode("Pa$$w0rd"));
+        userRepository.save(mock_user);
+
+        Long id = Long.valueOf(1);
```

Comment:
If other test case also create user in database and is running, this might not necessarily return the ID of the user that you have created in L40